test_runner/src/main/kotlin/ftl/run/AndroidTestRunner.kt

```diff
@@ -19,71 +19,66 @@ import kotlinx.coroutines.coroutineScope
 
 object AndroidTestRunner {
 
-    suspend fun runTests(androidArgs: AndroidArgs): Pair<MatrixMap, List<List<String>>> = coroutineScope {
-        val (stopwatch, runGcsPath) = GenericTestRunner.beforeRunTests(androidArgs)
+    suspend fun runTests(args: AndroidArgs): Pair<MatrixMap, List<List<String>>> = coroutineScope {
+        val (stopwatch, runGcsPath) = GenericTestRunner.beforeRunTests(args)
 
         // GcAndroidMatrix => GcAndroidTestMatrix
         // GcAndroidTestMatrix.execute() 3x retry => matrix id (string)
-        val androidDeviceList = GcAndroidDevice.build(androidArgs.devices)
+        val androidDeviceList = GcAndroidDevice.build(args.devices)
 
         val jobs = arrayListOf<Deferred<TestMatrix>>()
-        val runCount = androidArgs.repeatTests
+        val runCount = args.repeatTests
         val shardCounter = ShardCounter()
-        val history = GcToolResults.createToolResultsHistory(androidArgs)
-        val apks = resolveApks(androidArgs, runGcsPath)
-        val allTestShardChunks: MutableList<List<String>> = mutableListOf()
-
-        apks.forEach { apk ->
+        val history = GcToolResults.createToolResultsHistory(args)
+        val appTestApks = listOf(AppTestPair(app = args.appApk, test = args.testApk)) + args.additionalAppTestApks
+        val allTestShardChunks: List<List<String>> = appTestApks.map { localApk ->
```

Comment:
Do you think we could change variable name localApk? It is actually Pair not single path. What do you think?